app/components/syntax-highlighted-diff.ts

```diff
@@ -195,7 +253,7 @@ export default class SyntaxHighlightedDiffComponent extends Component<Signature>
 }
 
 declare module '@glint/environment-ember-loose/registry' {
-  export default interface Registry {
+  export default interface Registry extends EmberConcurrencyRegistry {
```

Comment:
Can we place this in `glint.d.ts` instead? Should be a global thing, not specific to `syntax-highlighted-diff`

Reply:
I believe this is needed so that `perform` helper can be used in the template

Reply:
Will try to make this global